### PR TITLE
Set adaptive icon background to black and update readme

### PR DESCRIPTION
**Pull request by jhw866**
Fix adaptive icon background by making the background all black.

I was only able to test on an android emulator using the circle icon. I can't seem to change the shape of the icons from the system settings in the emulator so it would be helpful to double check with some testing. Resolves issue #13 

I also updated the readme to reflect the system requirements for contributing. Apparently you will need to use AS 3.4 to use the latest version of gradle.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,6 +17,10 @@ By leveraging webviews as much as possible, we can smoothly sync up with our web
 
 We expect contributors to abide by our underlying [code of conduct](./CODE_OF_CONDUCT.md). All conversations and discussions on GitHub (issues, pull requests) and across dev.to must be respectful and harassment-free.
 
+### System Requirements
+
+You will need to have Android Studio 3.4 or up installed.
```
Comment (maestromac) [inline README.md]: This is a nice touch!

**Comment by jhw866** [comment]
> Thanks for this PR! The logo is slightly off though. Please use [this dev logo](https://user-images.githubusercontent.com/15793250/56824610-f761bd80-6824-11e9-8fbc-0255a54606bf.png) for the asset with [68% scaling](https://user-images.githubusercontent.com/15793250/56824714-3abc2c00-6825-11e9-86c7-d487dc6a79cc.png).

Would that asset replace `ic_launcher-web.png`?

**Comment by maestromac** [comment]
@jhw866 i could be wrong but I think Android studio uses that asset generates to `ic_launcher-web.png`

**Comment by jhw866** [comment]
@maestromac interesting. I didn't know that! Can confirm: https://stackoverflow.com/questions/13895390/what-does-the-ic-launcher-web-png-in-my-project-root-do

**Pull request merged**